src/modules/scoreApi.js

```diff
@@ -0,0 +1,46 @@
+class Score {
+  constructor(user, score) {
+    this.user = user;
+    this.score = score;
+  }
+
+    scoreData = [];
+
+    url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/:id/scores/:id';
+
+    getScore = () => {
+      const recentScore = document.querySelector('.recent-score-group');
+      recentScore.innerHTML = this.scoreData.map((element, index) => `<li class=${index % 2 !== 0 ? 'list-gray' : 'list-white'}>${element.user} : ${element.score}</li>`).join('');
+    }
+
+    fetchScore = async () => {
+      try {
+        const data = await fetch(this.baseUrl);
+        const res = await data.json();
+        const displayData = res.result.map((item) => item);
+        this.scoreData = displayData;
+        return this.getScore();
+      } catch (err) { return err; }
+    };
```

Comment:
- [ ] please make sure that scores are created and retrieved successfully  from the API
![image](https://user-images.githubusercontent.com/61790587/173640330-105ef1b0-dc31-46c1-88b0-f8b4e74e7b7e.png)

Reply:
ok thanks for helpful advice  I will retrieve successfully from the API